catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageWebConverter.java

```diff
@@ -0,0 +1,1760 @@
+
+    private static final String EMAIL_ADDRESS_KEY = "EmailAddress";
+
+    private static final String PARENT_KEY = "parent";
+
+    private static final String PERSON_FIRST_NAME = "firstName";
+
+    private static final String PERSON_MIDDLE_NAME = "middleName";
+
+    private static final String PERSON_LAST_NAME = "lastName";
+
+    private static final String PERSON_NAME_KEY = "PersonName";
+
+    private static final String PERSON_KEY = "Person";
+
+    private static final String ASSOCIATION_TYPE = "associationType";
+
+    private static final String ASSOCIATION_SOURCE_OBJECT = "sourceObject";
+
+    private static final String ASSOCIATION_TARGET_OBJECT = "targetObject";
+
+    private static final String ASSOCIATION_KEY = "Association";
+
+    private static final String EXT_ID_IDENTIFICATION_SCHEME = "identificationScheme";
+
+    private static final String EXT_ID_REGISTRY_OBJECT = "registryObject";
+
+    private static final String EXT_ID_VALUE = "value";
+
+    private static final String EXTERNAL_IDENTIFIER_KEY = "ExternalIdentifier";
+
+    private static final String EXTRINSIC_OBJECT_CONTENT_VERSION = "contentVersion";
+
+    private static final String EXTRINSIC_OBJECT_OPAQUE = "opaque";
+
+    private static final String EXTRINSIC_OBJECT_MIME_TYPE = "mimeType";
+
+    private static final String EXTRINSIC_OBJECT_KEY = "ExtrinsicObject";
+
+    private static final String LID_KEY = "Lid";
+
+    private static final String STATUS_KEY = "Status";
+
+    private static final String BINDING_ACCESS_URI = "accessUri";
+
+    private static final String BINDING_SERVICE = "service";
+
+    private static final String BINDING_TARGET_BINDING = "targetBinding";
+
+    private static final String SPECIFICATION_LINK_SERVICE_BINDING = "serviceBinding";
+
+    private static final String SPECIFICATION_LINK_SPECIFICATION_OBJECT = "specificationObject";
+
+    private static final String SPECIFICATION_LINK_USAGE_DESCRIPTION = "usageDescription";
+
+    private static final String SPECIFICATION_LINK_KEY = "SpecificationLink";
+
+    private static final String SPECIFICATION_LINK_USAGE_PARAMETERS = "usageParameters";
+
+    private static final String PRIMARY_CONTACT = "primaryContact";
+
+    private static final String ADDRESS_STREET_NUMBER = "streetNumber";
+
+    private static final String EMAIL_TYPE = "type";
+
+    private static final String PHONE_COUNTRY_CODE = "countryCode";
+
+    private static final String PHONE_TYPE = "phoneType";
+
+    private static final String CLASSIFICATION_NODE = "classificationNode";
+
+    private static final String CLASSIFIED_OBJECT = "classifiedObject";
+
+    private static final String CLASSIFICATION_SCHEME = "classificationScheme";
+
+    private static final String NODE_REPRESENTATION = "nodeRepresentation";
+
+    private static final String CLASSIFICATION_KEY = "Classification";
+
+    private RegistryPackageWebConverter() {
+    }
+
+    /**
```

Comment:
Make sure to run the formatter on this class. The comments' format is off for some reason.